### PR TITLE
added brandName component

### PR DESCRIPTION
**Pull request by ayazsaifi**

### CODE REVIEW

File: src/components/LogoMaker/BrandName/data.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const data = () => ({
+    logoText : JSON.parse(window.localStorage.getItem('localStorage')|| "{}").companyName
+}) ; 
+
+export default data ;
```
Comment (sahilbathla) [inline src/components/LogoMaker/BrandName/data.js]: new line

File: src/components/LogoMaker/BrandName/data.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const data = () => ({
+    logoText : JSON.parse(window.localStorage.getItem('localStorage')|| "{}").companyName
```
Comment (sahilbathla) [inline src/components/LogoMaker/BrandName/data.js]: let's move this to service, and keep the name `logoData`

File: src/components/LogoMaker/BrandName/methods.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const methods = {
+    saveText() {
+        let companyName = this.logoText;
```
Comment (sahilbathla) [inline src/components/LogoMaker/BrandName/methods.js]: const

File: src/components/LogoMaker/BrandName/methods.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+        const localStorageData = JSON.parse(window.localStorage.getItem('localStorage') || "{}");
```
Comment (sahilbathla) [inline src/components/LogoMaker/BrandName/methods.js]: use same service

File: src/components/LogoMaker/BrandName/methods.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+        window.localStorage.setItem('localStorage', JSON.stringify({...localStorageData, companyName}));
```
Comment (sahilbathla) [inline src/components/LogoMaker/BrandName/methods.js]: use the same service/model

```
class LogoModel {
     all(): {}
     find(id): {}
     update(id, data): {}
}
```

File: src/components/LogoMaker/BrandName/methods.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+    }
+}
+
+export default methods
```
Comment (sahilbathla) [inline src/components/LogoMaker/BrandName/methods.js]: new line

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 7]
```diff
@@ -0,0 +1,205 @@
+<template>
+<div>
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: indent

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+  <v-container class="icon-select-component pt-12 mt-12" fluid>
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: `icon-select-component` > `logo-icon`. Also remove dependency from bootstrap is possible

Comment (sahilbathla) [reply src/components/LogoMaker/LogoIcon/LogoIcon.vue]: what does fluid do?

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+    <v-row >
+      <v-col md="4">
+        <div style="position: fixed;" :style="{'width': width}">
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: no inline styling

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 10]
```diff
@@ -0,0 +1,205 @@
+<template>
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: index.vue

Comment (sahilbathla) [reply src/components/LogoMaker/LogoIcon/LogoIcon.vue]: filename

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 11]
```diff
@@ -0,0 +1,205 @@
...
+  <v-container class="icon-select-component pt-12 mt-12" fluid>
+    <v-row >
+      <v-col md="4">
+        <div style="position: fixed;" :style="{'width': width}">
+          <v-select
+            v-model="logoIcon"
+            :items="iconItems"
+            item-text="imageName"
+            item-value="image"
+            label="Select icon for your logo"
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: move all messages

to `messages.json`

Require the file here

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 12]
```diff
@@ -0,0 +1,205 @@
...
+          >
+          </v-select>
+          <div class="icon-select-component__preview-dark">
+            <img class="icon-select-component__preview-icon"  :src="logoIcon" alt="">
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: alt should never be empty

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue (hunk 10 above)
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: Seperate out, template, css & js

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 13]
```diff
@@ -0,0 +1,205 @@
...
+import image1 from "../../../assets/logo.svg"
+
+export default {
+  name: 'HelloWorld',
+    data: () => ({
+      logoIcon: JSON.parse((window.localStorage.getItem('localStorage') || "{}")).icon || image1,
+      opacity: 0,
+      selectedIcon: JSON.parse((window.localStorage.getItem('localStorage'))).iconId || 1,
+      width: '33%',
+      iconItems:[
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: load from json

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 14]
```diff
@@ -0,0 +1,205 @@
...
+      position: relative;
+      width: 100%;
+      height: 35vh;
+      text-align: center;
+    }
+
+    .icon-select-component{
+        &__preview-dark{
+          @extend %textPreview;
+          background: #333;
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: _colors mixin

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 15]
```diff
@@ -0,0 +1,205 @@
...
+          text-align: center;
+          border-radius: 8px 8px 0 0 ;
+        }
+        &__preview-light{
+          @extend %textPreview;
+          background: #fdfdfd;
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: _colors mixin

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 16]
```diff
@@ -0,0 +1,205 @@
...
+          text-align: center;
+          border: .5px solid #c4c4c4;
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: _colors mixin

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 17]
```diff
@@ -0,0 +1,205 @@
...
+          border-radius: 0 0 8px 8px;
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: use $spacer * 2, talk to @joshimanisha

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 18]
```diff
@@ -0,0 +1,205 @@
...
+        }
+        &__preview-icon{
+            width: 25%;
+            position: relative;
+            top: 50%;
+            transform: translateY( -50%);
+        }
+        &__font-selection-cards{
+          color: #333;
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: _colors

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 19]
```diff
@@ -0,0 +1,205 @@
...
+          text-align: center;
+          height: 120px;
+          border-radius: 8px;
+          cursor: pointer;
+          border: .5px solid #c4c4c4c4;
+        }
+        &__font-selection-cards:hover{
+          background: #333;
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: _colors everywhere

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 20]
```diff
@@ -0,0 +1,205 @@
...
+        }
+        &__icon-selection-cards-indicators{
+          width: 1%;
+          height: 119px;
+          background-image: linear-gradient(to bottom , #FF5F04, #FF9D42 );
+          position: absolute;
+          border-radius: 8px 0 0 8px;
+        }
+    }
+</style>
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: new line

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 21]
```diff
@@ -0,0 +1,205 @@
...
+        }
+        &__font-selection-cards:hover{
+          background: #333;
+          border: 0 solid #c4c4c4;
+          transition: .25s;
+        }
+        &__icon-selection-cards-indicators{
+          width: 1%;
+          height: 119px;
+          background-image: linear-gradient(to bottom , #FF5F04, #FF9D42 );
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: _colors

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue [hunk 22]
```diff
@@ -0,0 +1,205 @@
...
+          border: .5px solid #c4c4c4c4;
+        }
+        &__font-selection-cards:hover{
+          background: #333;
+          border: 0 solid #c4c4c4;
+          transition: .25s;
+        }
+        &__icon-selection-cards-indicators{
+          width: 1%;
+          height: 119px;
```
Comment (sahilbathla) [inline src/components/LogoMaker/LogoIcon/LogoIcon.vue]: what is 119

File: src/plugins/vuetify.js [hunk 23]
```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
+import Vuetify from 'vuetify/lib';
+
+Vue.use(Vuetify);
+
+export default new Vuetify({
```
Comment (sahilbathla) [inline src/plugins/vuetify.js]:
```suggestion
export default new Vuetify({});
```

File: src/router.js [hunk 24]
```diff
@@ -0,0 +1,18 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import HelloWorld from './components/HelloWorld'
+import LogoMaker from './components/LogoMaker/LogoMaker'
+import BrandName from './components/LogoMaker/BrandName/BrandName'
+
+Vue.use(VueRouter);
+
+const routes = [
+    { path: '/', component: HelloWorld },
```
Comment (sahilbathla) [inline src/router.js]: remove

File: src/router.js [hunk 25]
```diff
@@ -0,0 +1,18 @@
...
+    { path: '/company-name', component: BrandName },
+    { path: '/logo-maker', component: LogoMaker }
+]
+
+export default new VueRouter({
+    routes
+})
+
```
Comment (sahilbathla) [inline src/router.js]: extra new line

File: vue.config.js [hunk 26]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  "transpileDependencies": [
+    "vuetify"
+  ]
+}
```
Comment (sahilbathla) [inline vue.config.js]: new line

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue (hunk 8 above)
Comment (ayazsaifi) [reply src/components/LogoMaker/LogoIcon/LogoIcon.vue]: thats not bootsrap i am using vuetify

Comment (ayazsaifi) [reply src/components/LogoMaker/LogoIcon/LogoIcon.vue]: fluid is to remove padding and margin

File: src/components/LogoMaker/LogoIcon/LogoIcon.vue (hunk 12 above)
Comment (ayazsaifi) [reply src/components/LogoMaker/LogoIcon/LogoIcon.vue]: i will fill this with real data

**Pull request merged**